api/http/api_inventory.go

```diff
@@ -593,8 +598,16 @@ func (i *inventoryHandlers) PatchDeviceAttributesInternalHandler(
 		u.RestErrWithLog(w, r, l, err, http.StatusBadRequest)
 		return
 	}
-	for _, a := range attrs {
+	for i, a := range attrs {
 		a.Scope = r.PathParam("scope")
+		if a.Name == checkInTimeParamName {
```

Comment:
maybe check both scope (system) and name?

Reply:
done